package.json

```diff
@@ -3,18 +3,27 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "db:migrate": "node postgrator.js",
+    "db:migrate:test": "NODE_ENV=test node postgrator.js",
+    "db:createMigration": "node ./lib/createMigrations.js",
```

Comment:
If you're going for alphabetized here, line 8 should be above 6.

Reply:
Nice catch!